desktop/cmp/input/DateInput.ts

```diff
@@ -439,7 +439,7 @@ const cmp = hoistCmp.factory<DateInputProps & {model: DateInputModel}>(
                 enforceFocus: false,
                 modifiers: props.popoverModifiers,
                 position: props.popoverPosition ?? 'auto',
-                boundary: props.popoverBoundary ?? 'viewport',
+                boundary: props.popoverBoundary ?? 'clippingParents',
```

Comment:
see comment above  on this-- do we use clippingParents elsewhere?

Reply:
No, `clippingParents` is the popover default for `boundary`, but I believe this should actually be `rootBoundary: props.popoverBoundary ?? 'viewport'` based on the migration wiki you found. That being said, it's currently not working, and I don't believe we've seen any issues?  Will look into this more.